File: C#/0832. Flipping an Image.cs
public class Solution {
    public int[][] FlipAndInvertImage(int[][] A) {
        for(int i=0;i<A.Length;i++){
            A[i]=ReverseArray(A[i]);
        }
        for(int i=0;i<A.Length;i++){
            A[i]=Flip(A[i]);
        }
        return A;
    }
    public int[] ReverseArray(int[] nums){
        int left=0;
        int right=nums.Length-1;
        while(left<right){
            int tmp=nums[left];
            nums[left]=nums[right];
            nums[right]=tmp;
            left+=1;
            right-=1;
        }
        return nums;
    }
    public int[] Flip(int[] nums){
        for(int i=0;i<nums.Length;i++){
            nums[i]=nums[i]^1;
        }
        return nums;
    }
}

File: archives/leetcode2/0832. Flipping an Image.py
class Solution:
    def flipAndInvertImage(self, A: 'List[List[int]]') -> 'List[List[int]]':
        k=0
        while k<len(A):
            A[k]=A[k][::-1]
            i=0
            while i<len(A[k]):
                if A[k][i]==1:
                    A[k][i]=0
                else:
                    A[k][i]=1
                i+=1
            k+=1
        return A
